readme.md

```diff
@@ -170,7 +170,7 @@ For advanced JSON usage, check out the [`parseJson`](documentation/2-options.md#
 
 |                       | `got`               | [`request`][r0]    | [`node-fetch`][n0]   | [`ky`][k0]               | [`axios`][a0]      | [`superagent`][s0]     |
 |-----------------------|:-------------------:|:------------------:|:--------------------:|:------------------------:|:------------------:|:----------------------:|
-| HTTP/2 support        | :heavy_check_mark:¹ | :x:                | :x:                  | :x:                      | :x:                | :heavy_check_mark:\*\* |
+| HTTP/2 support        | :heavy_check_mark:¹ | :x:                | :heavy_check_mark:   | :x:                      | :x:                | :heavy_check_mark:\*\* |
```

Comment:
AFAIK it doesn't support HTTP/2 yet

https://github.com/node-fetch/node-fetch/issues/342

Reply:
sorry, replied for the wrong comment :(